graylog2-web-interface/src/util/URLUtils.ts

```diff
@@ -110,6 +110,10 @@ const URLUtils = {
 
     return acceptedProtocols.includes(url.protocol);
   },
+  urlQueryFromObject(object) {
+    return Object.entries(object)
+      .reduce((res, [key, value], i) => `${res}${i === 0 ? '' : '&'}${key}=${value}`, '');
+  },
```

Comment:
We probably do not need to create a new function for this, instead you can use the `PaginatedURL` function.